perceval/backends/core/stackexchange.py

```diff
@@ -279,6 +254,43 @@ def get_questions(self, from_date):
                                   nquestions,
                                   tquestions)
 
+    @staticmethod
+    def sanitize_for_archive(url, headers, payload):
+        """Sanitize the URL, headers and payload of a HTTP request before storing/retrieving items
+
```

Comment:
Add a `:returns` not specifying what this method returns. I think it would be useful to add paragraph, in the description of the method, about what parts are satanized and in which way.

This comment is applied to the rest of the backends which override this method.